ESMF/GOCART2G_GridComp/DU2G_GridComp/DU2G_StateSpecs.rc

```diff
@@ -7,27 +7,39 @@ category: IMPORT
 #----------------------------------------------------------------------------------------
      NAME   | UNITS  | DIMS | VLOC| COND | LONG NAME
 #----------------------------------------------------------------------------------------
- DU_SRC     | 1      | xy   | N   | scheme == 'ginoux'  | erod - dust emissions
- DU_CLAY    | 1      | xy   | N   | scheme == 'fengsha' | volume_fraction_of_clay_in_soil
- DU_SAND    | 1      | xy   | N   | scheme == 'fengsha' | volume_fraction_of_sand_in_soil
- DU_SILT    | 1      | xy   | N   | scheme == 'fengsha' | volume_fraction_of_silt_in_soil
+ DU_SRC     | 1      | xy   | N   |      | erod - dust emissions
+ DU_Z0      | 1      | xy   | N   |      | aerodynamic_surface_roughness_for_aeolian_processes
+ DU_GVF     | 1      | xy   | N   |      | GVF
+ DU_SAND    | 1      | xy   | N   |      | volume_fraction_of_sand_in_soil
+ DU_SILT    | 1      | xy   | N   |      | volume_fraction_of_silt_in_soil
+ DU_CLAY    | 1      | xy   | N   |      | volume_fraction_of_clay_in_soil
  DU_RDRAG   | m-1    | xy   | N   | scheme == 'fengsha' | drag_partition
  DU_SSM     | 1      | xy   | N   | scheme == 'fengsha' | sediment_supply_map
  DU_UTHRES  | m s-1  | xy   | N   | scheme == 'fengsha' | surface_dry_threshold_velocity
  FRSNOW     | 1      | xy   | N   | scheme == 'fengsha' | surface_snow_area_fraction
  SLC        | 1      | xy   | N   | scheme == 'fengsha' | liquid_water_content_of_soil_layer
+ DU_TEXTURE | 1      | xy   | N   |      | soil_texture
+ DU_VEG     | 1      | xy   | N   |      | vegetation_type
+ FRLAKE     | 1      | xy   | N   |      | fraction_of_lake
+ FRLAND     | 1      | xy   | N   |      | fraction_of_land
+ ASNOW      | 1      | xy   | N   |      | snow_covered_fraction_of_land
  WET1       | 1      | xy   | N   |      | surface_soil_wetness
  LWI        | 1      | xy   | N   |      | land-ocean-ice_mask
- FRLAKE     | 1      | xy   | N   |      | fraction_of_lake
+ TROPP      | Pa     | xy   | N   |      | tropopause_pressure_based_on_blended_estimate
```

Comment:
Of these, `U10N`, `V10N`, and `WCSF` seem to be K14 related.

`TROPP` though, seems to be GEOS endemic and it seems to come from `tropovars` in https://github.com/GEOS-ESM/GMAO_Shared/blob/main/GEOS_Shared/tropovars.F90. I think `TropP3` is eventually `TROPP_BLENDED`

Reply:
@mathomp4 - Thank you, this is helpful. We could implement `tropovars` in UFS-Aerosols to compute `TROPP` inline, but it requires potential vorticity as an input and this field is not available at this time.

Reply:
Hmm. Well, you are beyond my ken. Maybe @amdasilva or someone with science knowledge could help come up with a tropopause pressure that could be used as an approximate proxy? (Also, I have no idea if that is *actually* where TROPP_BLENDED comes from, but I think it might from my search.)

Reply:
If I understand it correctly, a "blended" tropopause pressure is determined "blending" the tropopause k-levels estimated by looking at temperature (`ktherm`) and potential vorticity (`kept`) profiles:
```
kblend = max( ktherm,kepv )
```
We could use other methods to compute the pressure at the tropopause that do not require the potential vorticity (see, for example, [Reichler _et al._, 2003](https://doi.org/10.1029/2003GL018240)). However, I don't know how they would compare with the blended method and this seems to introduce an uncomfortably high degree of uncertainty at this stage.

Reply:
Definitely needs a comment from @amdasilva

Reply:
@rmontuoro There is another possibility, maybe you don't care? If I look at `Process_Library/GOCART2G_Process.F90`, `tropp` is only used to calculate `stexttau` and `stscatau`, e.g.,:

https://github.com/GEOS-ESM/GOCART/blob/c66c311ef2db1cac8cda13a346bfb6ce7f93f5c3/Process_Library/GOCART2G_Process.F90#L3585-L3591

So, if UFS doesn't need to know about "stratospheric ext. AOT at 550 nm" or "stratospheric sct. AOT at 550 nm" maybe it's a throwaway? Just fill it with something that won't cause a "divide-by-zero" and move along?

Reply:
@mathomp4 - good catch! It is hard to make a decision on this right now since we are still discussing how aerosol-radiation feedback will be introduced in the UFS using GOCART. We might need to provide `stexttau` to the radiation scheme running in the atmospheric model. Also, I would like to preserve the ability to compute these diagnostics in UFS-Aerosols. Nevertheless, your suggestion is good and could be a starting point.

Reply:
One other option would be to add a procedure computing the atmospheric pressure at the tropopause to the process library. This would help achieve consistency across different applications.